/**:
  ros__parameters:
    keep_last_trajectory_s: 0.5 # [s]
    nearest_dist_threshold_m: 1.5 # [m]
    nearest_yaw_threshold_rad: 1.0 # [rad]
    target_pull_out_speed_mps: 1.0 # [mps]
    target_pull_out_acc_mps2: 1.0 # [mps2]
    max_speed_mps: 8.33 # [mps] [2.77 mps=10 kmph], [4.16 mps=15 kmph], [8.33 mps=30 kmph]
    max_lateral_accel_mps2: 1.5 #[mps2]
    spline_interpolation_resolution_m: 0.5 # [m]
    backward_trajectory_extension_m: 5.0 # [m]
    use_akima_spline_interpolation: true
    smooth_trajectories: true
    limit_speed: true
    limit_lateral_acceleration: true
    set_engage_speed: false
    fix_invalid_points: true
    smooth_velocities: false
    publish_last_trajectory: false
    keep_last_trajectory: false
    extend_trajectory_backward: true
